src/main/webapp/app/exam/participate/summary/exam-participation-summary.component.scss

```diff
@@ -0,0 +1,6 @@
+@import 'node_modules/bootstrap/scss/bootstrap';
```

Comment:
I'm sorry to be that guy again, but we need to avoid bootstrap imports. I know this is still done in one or the other component but it's really a bad idea.
Afaik such imports cause a very bloated file, plus this fresh import does not have access to theme variables and other various settings we usually pass into bootstrap.
I think it's fine to just include the class in the HTML if you need it.

Reply:
Already fixed in one commit earlier ;)

Reply:
lol
:D

Reply:
> Afaik such imports cause a very bloated file, plus this fresh import does not have access to theme variables and other various settings we usually pass into bootstrap.

Yes i noticed some issues with the button color and reverted that change :+1: